data retrieval/src/main/java/de/tinf15b4/kino/retrieval/tmdb/TmdbDataRetriever.java

```diff
@@ -0,0 +1,193 @@
+        List<MediaCreditCrew> crew = moviesInstance.getMovieCredits(mi.getId()).getCrew();
+
+        String result = "";
+
+        List<MediaCreditCrew> list = crew.stream()
+                .filter(c -> c.getDepartment().equals("Directing") && c.getJob().equals("Director"))
+                .collect(Collectors.toList());
+
+        if (list.size() > 0) {
+            Iterator<MediaCreditCrew> i = list.iterator();
+            while (i.hasNext()) {
+                MediaCreditCrew item = i.next();
+                result += item.getName() + (i.hasNext() ? ", " : "");
+            }
+            return result;
+        }
+        return "Keine Angabe";
+    }
+
+    private Genre getGenre(MovieInfo mi) {
+        // TODO: Bei Gelegenheit umstellen dass ein Film mehere Genres haben
+        // kann
+        List<com.omertron.themoviedbapi.model.Genre> genres = mi.getGenres();
+        if (genres.size() > 0) {
+            String name = genres.get(0).getName();
+
+            switch (name) {
+            case "Komödie":
+                name = "Komoedie";
+                break;
+            case "Science Fiction":
+                name = "ScienceFiction";
+                break;
+            case "TV-Film":
+                name = "TvFilm";
+            }
+
+            return Genre.valueOf(name);
+        }
+        return Genre.Unbekannt;
+    }
+
+    private AgeControl getAgeControl(MovieInfo mi) throws MovieDbException {
+        // Get all Release Date Information
+        List<ReleaseDates> releases = moviesInstance.getReleaseDates(mi.getId()).getResults();
+
+        // Get Germany Release Date Information
+        Optional<ReleaseDates> germany = releases.stream().filter(r -> r.getCountry().equals("DE")).findFirst();
+
+        if (germany.isPresent()) {
+            // Get Cinema Release Information
+            Optional<ReleaseDate> cinemaRelease = germany.get().getReleaseDate().stream()
+                    .filter(r -> r.getType() == ReleaseType.THEATRICAL).findFirst();
+
+            if (cinemaRelease.isPresent()) {
+                String age = cinemaRelease.get().getCertification();
+
+                if (!age.equals("")) {
+                    return AgeControl.valueOf("USK" + age);
+                }
+            }
+        }
+        return AgeControl.UNBEKANNT;
+    }
+    //
+    // }
+    //
+    // private static String readUrl(String urlString) throws Exception{
+    // BufferedReader reader = null;
+    // try{
+    // URL url = new URL(urlString);
+    // reader = new BufferedReader(new InputStreamReader(url.openStream()));
+    // StringBuffer buffer = new StringBuffer();
+    // int read;
+    // char[] chars = new char[1024];
+    // while((read = reader.read(chars)) != -1){
+    // buffer.append(chars, 0, read);
+    // }
+    // return buffer.toString();
+    // } finally {
+    // if (reader != null){
+    // reader.close();
+    // }
+    // }
```

Comment:
Why is this commented out? Just delete it if its not needed, otherwise add some comment why we need to keep it